CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 Changelog
 =========
 
-* **2015-14-02**: implement sitemap generation
-* **2015-14-02**: [BC BREAK] Changed method visibility from private to public of SeoPresentation#getSeoMetadata()
+* **2015-02-24**: Configuration for content_key moved to content_listener section, and its now possible to disable
+  The content listener by setting cmf_seo.content_listener.enabled: false
+* **2015-02-14**: implement sitemap generation
+* **2015-02-14**: [BC BREAK] Changed method visibility from private to public of SeoPresentation#getSeoMetadata()
```

Comment:
@dbu yes this is fine we havent started using this just yet so no changes will be necessary on our end